### PR TITLE
set up basic testing with detox and jest

### PR DESCRIPTION
**Pull request by s3687636**
first testing:
- check if the app open with "home page"

### CODE REVIEW

File: client/ios/Podfile.lock [hunk 1]
```diff
@@ -182,6 +182,8 @@ PODS:
     - React-cxxreact (= 0.61.5)
     - React-jsi (= 0.61.5)
   - React-jsinspector (0.61.5)
```
Comment (trungduong0103) [inline client/ios/Podfile.lock]: we don't need to commit Podfile.lock, the packages should be installed whenever we use yarn install and then go to ios folder to instal pods.

File: client/package.json [hunk 2]
```diff
@@ -32,12 +32,28 @@
     "@babel/runtime": "^7.8.7",
     "@react-native-community/eslint-config": "^0.0.7",
     "babel-jest": "^25.1.0",
```
Comment (trungduong0103) [inline client/package.json]: I can't comment on package-lock.json but we also don't commit package-lock.json, package.json is fine here

Comment (trungduong0103) [reply client/package.json]: Also we don't have to commit yarn.lock

**Comment by trungduong0103** [comment]
> first testing:
> 
> * check if the app open with "home page"

one other thing, please change it to client/testing_setup or something similar. It's just convention four our project.

File: client/ios/Podfile.lock (hunk 1 above)
Comment (s3687636) [reply client/ios/Podfile.lock]: ok

File: client/package.json (hunk 2 above)
Comment (s3687636) [reply client/package.json]: package.-lock.json will protect our developer from dependency confilcts

**Comment by s3687636** [comment]
the branch is already client-testing-setup.
client/testingsetup will make it a subbranch of client?

File: client/package.json (hunk 2 above)
Comment (trungduong0103) [reply client/package.json]: good to know. But we will use yarn, using npm cannot fix dependencies warning whenever a package goes bad, yarn updates our packages without having to use npm audit fix every time we encounter a vulnerability issue.

**Comment by trungduong0103** [comment]
> the branch is already client-testing-setup.
> client/testingsetup will make it a subbranch of client?

we can rename our branch, it's just that my past push request are client/branch_name, it just means this is a push request for client.

File: client/package.json (hunk 2 above)
Comment (trungduong0103) [reply client/package.json]: please remove your package-lock.json file and use yarn install instead, I hesitated from switching to yarn but after using it, I think that it's better than npm, just for the vulnerability fixes.

Comment (s3687636) [reply client/package.json]: removed. at commit e83182a

**Pull request merged**